frontend/src/OneBuilding/index.jsx

```diff
@@ -45,11 +46,12 @@ const styles = StyleSheet.create({
 });
 
 function OneBuilding({
-  onClickClose, targetBuildingId, getIconImage,
+  onClickClose, targetBuildingId, visible,
```

Comment:
```suggestion
  onClickClose, targetBuildingId, isVisible,
```

it's a boolean, please rename